client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/components/GetHelpPage.css

```diff
@@ -0,0 +1,79 @@
+.get-help-page-paper {
```

Comment:
https://github.com/Coursemology/coursemology2/blob/master/client/README.md#-only-use-tailwind-utilities-for-styling